engine/src/main/java/org/terasology/rendering/gltf/model/GLTFInterpolation.java

```diff
@@ -30,15 +30,17 @@
     LINEAR {
         @Override
         public void interpolate(Vector3f a, Vector3f b, float t, Vector3f out) {
-            out.x = a.x + t * (b.x - a.x);
-            out.y = a.y + t * (b.y - a.y);
-            out.z = a.z + t * (b.z - a.z);
+            a.lerp(b,t,out);
+//            out.x = a.x + t * (b.x - a.x);
+//            out.y = a.y + t * (b.y - a.y);
+//            out.z = a.z + t * (b.z - a.z);
```

Comment:
remove commented code and apply formatter.
```suggestion
            a.lerp(b, t, out);
```